sim/hunter/serpent_sting.go

```diff
@@ -66,8 +66,8 @@ func (hunter *Hunter) registerSerpentStingSpell() {
 			var result *core.SpellResult
 
 			if hunter.Talents.ImprovedSerpentSting != 0 {
-				baseDamage := 460 + 0.40*spell.RangedAttackPower(target)
-				result = spell.CalcDamage(sim, target, baseDamage*float64(hunter.Talents.ImprovedSerpentSting), spell.OutcomeRangedHitAndCrit)
+				baseDamage := (460 * 5) + 0.40*spell.RangedAttackPower(target)
+				result = spell.CalcDamage(sim, target, baseDamage*(0+float64(hunter.Talents.ImprovedSerpentSting)*0.15), spell.OutcomeRangedHitAndCrit)
```

Comment:
You already have `hunter.Talents.ImprovedSerpentSting*0.15` in the damage multiplier of the spell so this is double dipping here